Makefile

```diff
@@ -187,6 +191,23 @@ bundle: manifests kustomize
 		sed -i "s@containerImage: controller:latest@containerImage: $(IMG)@g" bundle/manifests/aerospike-kubernetes-operator.clusterserviceversion.yaml; \
 	fi
 
+# OPENSHIFT_VERSION has to be defined to build bundle for openshift platform
```

Comment:
Can the same bundle with these extra label work on regular kubernetes. If so we can always add openshift labels.

Reply:
It should work for non-openshift. I'll see and make the changes.

Reply:
Added Openshift labels to common image.